packages/cfpb-layout/src/molecules/card.less

```diff
@@ -1,27 +1,78 @@
+.u-card-link-text() {
+    .m-card_footer > span {
+        display: inline;
+        border-width: 0;
+        border-bottom-width: 1px;
+        border-color: @pacific;
+        border-style: dotted;
+        font-weight: 500;
+        color: @pacific;
+        text-decoration: none;
+    }
+
+    & > a:visited .m-card_footer > span {
+        color: @teal;
+        border-color: @teal !important;
+    }
+
+    &:hover {
+        .m-card_footer > span {
+            border-style: solid;
+            color: @pacific-dark;
+        }
+    }
+}
+
+.u-card-bottom-bar() {
+    // Bottom green hover bar.
+    &:after {
+        content: '';
+        position: absolute;
+        left: 0;
+        bottom: 1px;
+        height: 5px;
+        width: 100%;
+        background: @green;
+    }
+}
+
 .m-card {
-    width: 100%;
-    background: @white;
+    position: relative;
 
-    // Both regular and featured cards.
-    &:not(.m-card__featured),
+    &:not( .m-card__breakout ) {
+        & > a {
+            padding: unit( @grid_gutter-width / @base-font-size-px, em );
+        }
+    }
+
+    &:not( .m-card__breakout ):not( .m-card__highlight ):not( .m-card__topic ) {
+        background: @white;
+        border: 1px solid @gray-20;
+        border-bottom-width: 3px;
+    }
+
+    &:not( .m-card__featured ):not( .m-card__breakout ):not( .m-card__topic ):not( .m-card__highlight ) {
```

Comment:
L58 includes inner links, so instead of resetting the padding rule for links, I separated them. Some of this can be cleaned up when featured cards are removed.